src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.Id.Random.cs

```diff
@@ -3,10 +3,10 @@
     public partial class Activity
     {
         #region private
-        private string generateUniquePrefix()
+        private string generateInstancePrefix()
         {
             byte[] bytes = new byte[8];
-            s_random.Value.NextBytes(bytes);
+            s_random.NextBytes(bytes);
```

Comment:
`s_random` is a static Random instance?  Is there higher-level locking being done that ensures this Random instance isn't used concurrently from multiple threads?  Doing so is not thread-safe, and will likely corrupt the Random instance.